src/scripts/ads/vast/VASTIntegrator.js

```diff
@@ -324,13 +324,30 @@ VASTIntegrator.prototype._playSelectedAd = function playSelectedAd(source, respo
 
       player.trigger('vast.adStart');
 
-      playerUtils.once(player, ['ended', 'vast.adsCancel', 'vast.adSkip'], function (evt) {
+      player.on('ended', proceed);
```

Comment:
why don't you use 
```javascript
playerUtils.once(player, ['ended', 'vast.adsCancel', 'vast.adSkip'], function(evt){ ....
```
it will do the player.off automatically